### PR TITLE
Ignore MyPy errors for providers Amazon S3 operators

### PR DESCRIPTION
**Pull request by sunank200**
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

**Review by kaxil — COMMENTED** [review]
cc @uranusjr Can you take a look at this, is there a way around this, the `.pyi` files are causing the issue

File: airflow/providers/amazon/aws/operators/s3_file_transform.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 import warnings
 
-from airflow.providers.amazon.aws.operators.s3 import S3FileTransformOperator  # noqa
+from airflow.providers.amazon.aws.operators.s3 import S3FileTransformOperator  # type: ignore # noqa
```
Comment (kaxil) [inline airflow/providers/amazon/aws/operators/s3_file_transform.py:22]: This is caused by [s3.pyi](https://github.com/apache/airflow/blob/main/airflow/providers/amazon/aws/operators/s3.pyi) file

**Comment by eladkal** [comment]
I think all mypy issues were addressed already?

**Comment by potiuk** [comment]
YES!

**Pull request closed without merging**